docs/compression_algorithms/CompressWeights.md

```diff
@@ -302,42 +303,72 @@ Here is the word perplexity with data-free and data-aware mixed-precision INT4-I
         <td>11.87</td>
     </tr>
     <tr>
-        <td>meta-llama/llama-7b-chat-hf</td>
+        <td></td>
```

Comment:
We need baseline metrics for Wikitext as well. @andreyanufr, can you please add them?

Reply:
Can we also add a model footprint for consistency with the previous table? This is also the only differentiation factor that we can publically claim. @andreyanufr, please add it as well.

Reply:
Please also state where `all_layers` option was used if so.